cmd/sqlflow/main.go

```diff
@@ -51,12 +51,16 @@ func compile(cgName, sqlProgram, datasource string) (string, error) {
 
 func main() {
 	ds := flag.String("datasource", "", "database connect string")
+	logPath := flag.String("log", "", "path/to/log, e.g.: /var/log/sqlflow.log")
```

Comment:
This is not the entrypoint of `sqlflowserver`, maybe you can update the `cmd/sqlflowserver/main.go`

Reply:
Sure, this PR mainly implements the `pkg/log` and makes very few replacements.